pym/bob/input.py

```diff
@@ -15,6 +15,7 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 from . import BOB_VERSION
+from . import BOB_INPUT_HASH
```

Comment:
Should be merged into the previous line: `from . import BOB_VERSION, BOB_INPUT_HASH`